doc/navigating.rst

```diff
@@ -65,18 +71,46 @@ Using the Account Settings Window
 .. index:: account settings, user, password, Server URL
 
 Click **Settings** in the right-click menu to see a summary of your ownCloud 
-account settings. This shows which ownCloud account you are connected to (or accounts, if you have more than one) your 
+account settings, or left-click your systray icon. This shows which ownCloud 
+account you are connected to (or accounts, if you have more than one) your 
 quota status, and a window for managing your synchronization settings.
 
 .. image:: images/client6.png
    :alt: Account settings window
 
-At the top of the window are tabs for each configured sync account, and three others for Activity, General and Network settings.
+At the top of the window are tabs for each configured sync account, and three 
+others for Activity, General and Network settings. On your account tabs you 
+have the following features:
+
+* Connection status, showing which ownCloud server you are connected to, and 
+  your ownCloud username.
+* A **Remove Account** button, which deletes your account but does not delete 
+  your data files.
+* Used and available space on the server.
+* Current synchronization status.
+* **Add Folder Sync Connection** button, which is active only when you have 
+  removed synchronization on an account (see **Remove Sync** below).
+
+The little button with three dots that sits to the right of the sync status bar 
+offers three additional options:
+
+* Open Folder
+* Pause Sync
+* Remove Sync
+
+**Remove Sync** suspends all synchronization with an account without removing 
+the account. This is useful when you know you're not going to be able to 
+connect to your ownCloud server, for example when you're traveling, or when you 
+want to save bandwidth. When you're ready to resume synchronization click the 
+**Add Folder Sync Connection** button.
```

Comment:
I think is is ab it of a mixed description between Pause sync and Remove sync. If you are just going to be traveling or whatever and do not want the folder sync connection to update for a while, then Pause sync is the thing to do. You can Resume sync any time later.
If you do not want the folder sync connection any more, then Remove sync is the thing to do. The difficulty in describing the use of Remove sync is that it will not be used by 95% of people. If you have a single folder sync connection to sync the root of your server folders to some 1 place on your client system, then you probably do not want to touch anything.
If you have made multiple folder sync connections to sync various individual server folders to various individual places on the client, then you are going to use "Remove sync" and "Add folder sync connection" to setup and manage that.